### PR TITLE
CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mihok**
Finally! Solves issue #2

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## How to become a contributor and submit your own code
+
+### Contributing A Patch
+
+1. Submit an issue describing your proposed change to the repo in question.
+1. The [repo owners](OWNERS) will respond to your issue promptly.
+1. If instructed by the repo owners provide a short design document in a PR.
+1. Fork the desired repo, develop and test your code changes. Unit tests are required for most PRs.
+1. Submit a pull request.
```
Comment (teesloane) [inline CONTRIBUTING.md]: these are all listed `1` ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Contributing guidelines
+
+## How to become a contributor and submit your own code
+
+### Contributing A Patch
+
+1. Submit an issue describing your proposed change to the repo in question.
+1. The [repo owners](OWNERS) will respond to your issue promptly.
```
Comment (teesloane) [inline CONTRIBUTING.md]: > will respond as promptly as possible

thoughts?

File: CONTRIBUTING.md (hunk 1 above)
Comment (mihok) [reply CONTRIBUTING.md]: When u view it they render properly

File: CONTRIBUTING.md (hunk 2 above)
Comment (mihok) [reply CONTRIBUTING.md]: Copied from another open source project but I'm fine with it to start, doesn't mean we're building features promptly

**Comment by mihok** [comment]
What do you mean by style guide? (example?) also code of conduct I'm only familiar with for events not open source work, care to link an example?

File: CONTRIBUTING.md (hunk 1 above)
Comment (teesloane) [reply CONTRIBUTING.md]: lol

**Comment by teesloane** [comment]
style guide aka all the things that mihok nit picks when tees writes javascript for him ;)

[code of conduct generator](https://github.com/sindresorhus/conduct)

**Comment by mihok** [comment]
Thats what prettier and eslint are for ;)

**Comment by mihok** [comment]
Cool, ill add a CoC!

**Pull request merged**